<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Source Localization: EMG</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grpboxMethSel_">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>424</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Method Selection</string>
    </property>
    <widget class="QRadioButton" name="rdbtn_sLORETA">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>sLORETA</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpbox_MethSett">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>424</y>
      <width>341</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Method Settings</string>
    </property>
    <widget class="QComboBox" name="cbox_NumDipoles">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cBox_NumElec">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lbl_NumDipoles">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of dipoles per intervertebral section:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_NumElec">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of electrodes:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpbox_geom">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>234</y>
      <width>551</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Choose geometry</string>
    </property>
    <widget class="QRadioButton" name="rdbtn_GeomMeasur">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>170</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Use individual geometry</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbtn_AltGeom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>200</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Take alternative geometry</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledit_C7L5">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>90</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledit_InionC7">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ledit_NeckCircum">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_NeckCircum">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Neck cirumference (cm):</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_InionC7">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>140</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Inion-C7 distance (cm):</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_C7L5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>130</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>C7-L5 distance (cm):</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpBox_EMG">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>114</y>
      <width>551</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Select EMG data</string>
    </property>
    <widget class="QRadioButton" name="rdbtn_MultiPart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>150</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Multiple participants</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdbtn_SinglePart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>140</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Single participant</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_LoadData">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Data</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_DataPath">
     <property name="geometry">
      <rect>
       <x>179</x>
       <y>60</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Data path:</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl_Data">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>93</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Method">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>394</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Method</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_StartCalc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>544</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start Calculation</string>
    </property>
   </widget>
   <widget class="Line" name="line_DataMethod">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>364</y>
      <width>551</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_MethodVis">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>524</y>
      <width>551</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphic_INE">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>580</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphic_TUGraz_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>640</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_Feedback">
    <property name="geometry">
     <rect>
      <x>589</x>
      <y>653</y>
      <width>581</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="img_TUGraz">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Downloads/Logo_Graz.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img_INE">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Downloads/ine.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Feedback">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>Start</string>
    </property>
    <addaction name="actionLoad_new_file"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGet_more_information"/>
   </widget>
   <addaction name="menuStart"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_new_file">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Load new file</string>
   </property>
  </action>
  <action name="actionGet_more_information">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Get more information</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
